doc/ref_impl/cntt-ri2/chapters/chapter01.md

```diff
@@ -29,19 +29,41 @@ In order to promote the quick deployment of the RI and avoid duplicating effort
 
 <a name="1.1.3"></a>
 ### 1.1.3 Terminology
 
+Terminology in this document will follow [CNTT Terminology](../../../tech/glossary.md).
+
 <a name="1.2"></a>
 ## 1.2 Scope
 
+The scope of this document is as follows:
+1. To generate eco-system requirements for the establishment of the RI, including labs, tooling, installers, releases and automation requirements
+1. Provide detailed description file definition for installers, CNF features, and etc..
+1. Provide detailed lab criteria and operations that are generic enough to allow any environment to be used as the "lab".
+1. Provide an operational runbook for the RI, which includes detailed steps for the deployment and configuration of the RI into a chosen lab environment
+1. Provide detailed design for automation of deployment and testing, provide continuous integration and delivery pipeline for the RI
+1. Gap analysis with required actions for existing eco-system within adjacent community projects
+
 <a name="1.3"></a>
 ## 1.3 Reference Implementation Approach
 
-> Describe the approach taken in building the RI (e.g. combined infra/k8s LCM, or sepaarate, etc.)
+The approach taken for the RI has deliberately started by having separate automation stages for the infrastructure on which the RI will be installed, and the RI itself. Meaning, initially, the RI is not looking to have a single installer that can both build out virtual or physical machines **and** build out the Kubernetes and other components.
+
+The primary reason for this was to ensure the loose coupling between the two layers, to drive the concept that RI2 is a standalone platform that can (in theory at least) be deployable to any infrastructure, whether that be some physical machines in a lab, or virtual machines in a private or public cloud environment, for example. To combine the deployment of machines and Kubernetes platform with a single installer would potentially limit the locations on which the RI can be installed.
+
+However, that's not to say full end-to-end installers aren't welcome, but if used we must be clear on the limitations and compromises made when we document them in [Chapter 4](./chapter04.md).
 
 
 <a name="1.4"></a>
 ## 1.4 Relationship to other communities
+
+The main communities involved in driving requirements and development of this RI are:
+- **OPNFV - Cloud Native OVP aka OVP2.0**
+    - OPNFV, and CNOVP within it, owns the definition of the end-to-end framework (tooling, process, tests) used to verify and certify cloud native infrastructure and network functions
+    - This verification and certification process is the end goal for the Rx2 stream of documents; the RC2 is defining tests that will be used by vendors and others to validate conformance to the RA2 specification, with RI2 being a deployable reference for vendors to develop against
```

Comment:
terminology nit-pick: better refer to "conformance testing" instead of "certify / certification"
```suggestion
    - This verification process is the end goal for the Rx2 stream of documents; the RC2 is defining tests that will be used by vendors and others to validate conformance to the RA2 specification, with RI2 being a deployable reference for vendors to develop against
```

Reply:
As above - certification is valid as owned by CNOVP?

Reply:
Should this be "verification and validation"

Verification is typically accomplished through reviews, waltk-throughs and inspections. Thus, verification could be utilised to examine the manifests and compare them against the configuration specifications.

Validation is performed using testing.

Reply:
verification and validation works for me